### PR TITLE
Add theorems related to iscnrm3 and separated sets

### PR DESCRIPTION
**Pull request by zwang123**
# Added

* `iscnrm3` and `iscnrm4`: theorems related to completely normal spaces
    * And theorems used by these theorems, directly or indirectly.
* Theorems related to separated set: sepXXX 
    * And theorems used by these theorems, directly or indirectly.

# Fixed

* Usability of some theorems
    * Distint variables
    * setvar vs class
* etc

### CODE REVIEW

**Comment by zwang123** [comment]
`sspreima` is quite useful in my proof... Do I move it to main in order to use it?

**Comment by icecream17** [comment]
yes, if you use a theorem, it is useful, so it can be moved to main

edit: probably somewhere just after inpreima

**Review by icecream17 — APPROVED** [review]
mathbox spot check; optional typos; can also wait for others to comment to group changes

**Review by tirix — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -775019,6 +775426,400 @@ additional condition (deduction form).  See ~ ralbidc for a more
...
+  $( Lemma for ~ iscnrm3r .  The designed subspace is a subset of the original
+     set; the two sets are closed sets in the subspace.  (Contributed by Zhi
+     Wang, 5-Sep-2024.) $)
+  iscnrm3rlem3 $p |- ( ( J e. Top /\ ( S e. ~P U. J /\ T e. ~P U. J ) ) -> ( (
+  U. J \ ( ( ( cls ` J ) ` S ) i^i ( ( cls ` J ) ` T ) ) ) e. ~P U. J /\ ( ( (
+  cls ` J ) ` S ) \ ( ( cls ` J ) ` T ) ) e. ( Clsd ` ( J |`t ( U. J \ ( ( (
+  cls ` J ) ` S ) i^i ( ( cls ` J ) ` T ) ) ) ) ) /\ ( ( ( cls ` J ) ` T ) \ (
+  ( cls ` J ) ` S ) ) e. ( Clsd ` ( J |`t ( U. J \ ( ( ( cls ` J ) ` S ) i^i (
+  ( cls ` J ) ` T ) ) ) ) ) ) ) $=
+    ( ctop wcel cuni cpw wa ccl cfv cin cdif crest co ccld uniexg difssd elpwid
```
Comment (avekens) [inline set.mm:775605]: This theorem could be better structured (in  the future...).

**Review by avekens — APPROVED** [review]

**Pull request merged**